core/unit_test/TestUniqueToken.hpp

```diff
@@ -116,6 +132,23 @@ class TestUniqueToken {
       }
     }
 
+    // Count test for pull request #3260
+    {
+      constexpr int N = 1000000;
+      constexpr int R = 10;
```

Comment:
Consider making `R` a data member or comment that it has to match the definition in the functor above.